packages/grid/src/mouse-handlers/GridLinkMouseHandler.ts

```diff
@@ -94,85 +94,74 @@ class GridLinkMouseHandler extends GridMouseHandler {
       truncationChar
     );
```

Comment:
Call `getCachedTruncatedString` instead of `truncateToWidth` (this is the closest github will let me comment)